CONTRIBUTING.md

```diff
@@ -21,6 +31,106 @@ For general contribution and community guidelines, please see the [community rep
   compatible for those configurations. We expect at the very least that you will
   need Go modules support so you will at minimum need Go v1.11+.
 
+### Debug Using Delve
+
+![DebugFlow](/Users/Tamir.Zheleznyak/Documents/Code/conjur-authn-k8s-client/dev/DebugFlow.png)
+
+For development porupuses you may want to debug your code on K8S pod. There are the steps to do it:
+
+1. Make sure `delve` installed
+   `brew install delve`
+
+1. Make sure `kind` installed on your environment
+   
+   ```bash
+   curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.11.1/kind-linux-amd64
+   chmod +x ./kind
+   mv ./kind /usr/local/bin/kind
+   ```
+   
+2. Make sure you have `kubectl`
+   
+   ```bash
+   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+   sudo install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+   ```
+   
+2. Create kind cluster
+   `kind create cluster --config dev/kind.yaml`
+
+2. Build docker image for debugging with the `Dockerfile.Debug` running:
+   `docker build -f dev/Dockerfile.debug -t authn-k8s-client-debug:latest .`
+
+2. Load the docker image to your kind cluster
+   `kind load docker-image authn-k8s-client-debug:latest`
+
+7. Create and run a deployment with authn-k8s-client-debug:latest and your configuration. In your deployment please note these things (you may use this example)
+
+   1. Port for debugger (In my example its 40000)
+
+   2. volumeMounts and volume for 
```

Comment:
Not sure what this is trying to say. Maybe just `volumeMounts and volumes`?

Reply:
volumeMounts